XRTK-Core/Packages/com.xrtk.core/Runtime/Definitions/Controllers/Hands/HandControllerPoseDefinition.cs

```diff
@@ -4,30 +4,30 @@
 using UnityEngine;
 using XRTK.Definitions.Utilities;
 
-namespace XRTK.Definitions.Controllers.Simulation.Hands
+namespace XRTK.Definitions.Controllers.Hands
 {
     /// <summary>
-    /// A hand pose definition with recorded hand joint data.
-    /// Assign a collection of these poses to the data provider's profile to simulate the pose.
+    /// A hand controller pose definition with recorded hand joint data.
+    /// Defined hand poses can be recognized and trigger input actions.
     /// </summary>
-    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Input System/Simulated Hand Controller Pose Data", fileName = "SimulatedHandControllerPoseData", order = (int)CreateProfileMenuItemIndices.Input)]
-    public class SimulatedHandControllerPoseData : BaseMixedRealityProfile
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Input System/Hand Controller Pose Definition", fileName = "HandControllerPoseDefinition", order = (int)CreateProfileMenuItemIndices.Input)]
+    public class HandControllerPoseDefinition : BaseMixedRealityProfile
```

Comment:
Not sure this fits with the established naming patterns. Maybe `HandControllerPoseDefinitionProfile`? or just `HandControllerPoseProfile`?

I'm also a little concerned that this data structure is very similar to input actions. I would like to refine this a bit more and remove its reference from the `HandData` class as profiles should only be consumed by services and data providers.

Reply:
It's HandControllerPoseProfile now